### ISSUE TITLE
liste of carrier : wrong mention of the price

### ISSUE THREAD

**Issue by jdo17** [PrestaShop/PrestaShop#14522]
**Describe the bug**
purchase process step 3:
the carriers all display "free" for delivery costs. (liste of carriers)
However, as soon as a selection is made we have the price of delivery displayed in the summary insert aside of this list.

Ps:
The bug was noticed after the installation of the rc2. We have only configure one carrier with a simply price and two basket rules.

**To Reproduce**

**Screenshots**
If applicable, add screenshots or screenrecords to help explain your problem.

![6282880832241664-2](https://user-images.githubusercontent.com/29708024/60794079-41dab100-a169-11e9-89c5-1a73c098b40d.png)

**Additionnal information**
PrestaShop version: prestashop1-7-6-rc2
PHP version: N/A

**Comment by khouloudbelguith** [comment]
Hi @jdo17,

Thanks for your report.
I have the same issue with PS1.7.5.2 & PS1.7.6.0.
Steps to reproduce the issue:
1. Go to the BO => Shipping => Preferences => Set "Default carrier" to "My carrier
![image](https://user-images.githubusercontent.com/16067358/60797124-9d576f80-a166-11e9-9442-7c1c56dfbb24.png)
2. Create a cart rule "free shipping"
3. Go to the FO => Add some products to the cart 
4. Go to the checkout page => "Shipping method step" (My carrier is selected) => 
See the error in the checkout summary"

![image](https://user-images.githubusercontent.com/16067358/60798237-a8ab9a80-a168-11e9-9411-496effef8f95.png)

I’ll add this to the debug roadmap so that it’s fixed. If you have already fixed it on your end or if you think you can do it, please do send us a pull request!
Thanks!

**Comment by marionf** [comment]
@khouloudbelguith It seems to be a duplicate of #18533
Could you please verify and try to reproduce it on 1.7.7.x ?

**Comment by khouloudbelguith** [comment]
Hi @marionf,

I cannot check this ticket now, it is blocked by this issue:
https://github.com/PrestaShop/PrestaShop/issues/20523

Thanks!

**Comment by khouloudbelguith** [comment]
@marionf, yes it is fixed in the 1.7.7.x.
![image](https://user-images.githubusercontent.com/16067358/89793856-9d266200-db26-11ea-9416-8d8707b9e32e.png)

I will close the issue.

**Comment by khouloudbelguith** [comment]
Duplicate of #18533

**Comment by marionf** [comment]
Thank you @khouloudbelguith :+1:
